src/backend/distributed/utils/metadata_cache.c

```diff
@@ -34,6 +34,19 @@
 #include "utils/relmapper.h"
 #include "utils/syscache.h"
 
+#include "access/xact.h"
+
+
+/* state which should be cleared upon DROP EXTENSION */
+static bool extensionLoaded = false;
+static Oid distShardRelationId = InvalidOid;
+static Oid distShardPlacementRelationId = InvalidOid;
+static Oid distPartitionRelationId = InvalidOid;
+static Oid distPartitionLogicalRelidIndexId = InvalidOid;
+static Oid distShardLogicalRelidIndexId = InvalidOid;
+static Oid distShardShardidIndexId = InvalidOid;
+static Oid distShardPlacementShardidIndexId = InvalidOid;
```

Comment:
Maybe give them a prefix like cachedDistShardRelationId to make the purpose more clear and distinguish from the function names.